src/class-filters.php

```diff
@@ -13,9 +13,50 @@
  * The Filters class, used for defining available filters.
  */
 class Filters {
-  const DYNAMICS_CRM           = 'eightshift_forms/dynamics_info';
-  const ALLOWED_BLOCKS         = 'eightshift_forms/allowed_blocks';
-  const GENERAL                = 'eightshift_forms/general_info';
-  const PREFILL_GENERIC_MULTI  = 'eightshift_forms/prefill/multi';
-  const PREFILL_GENERIC_SINGLE = 'eightshift_forms/prefill/single';
+
```

Comment:
The same is as with actions. You could maybe have interface `Hooks` if this is being reused across the codebase.

Reply:
This is more for documentation / keeping list of all filters / actions in a single place (and ofc so that we don't have to write them as strings each time they're used).

Personally I like when documentation lists all available hooks (filters / actions) in a single place. This is the equivalent of that in code and can later be used to automatically generate documentation for each filter / action in a single place (I'd just add documentation as constant comment and run phpdocs. Or alterntively just copy all the comments to github wiki).

Having constants added to different places using `Hooks` interface would make compiling a list of available actions / filters much more difficult (as they'd be scattered about the codebase). Think about if documentation did this? You're looking for a list of available hooks to use but they're scattered on different pages.

Also, this is the same logic by which we for example keep all custom post types in a single folder. We group things by functionality (custom post types, custom taxonomies, blocks, etc), not by use context. Hence why all filters / actions are listed in the same place, the same way as all custom post types (for example) are listed in the same place.

Reply:
You can look at my response above. Keeping them in one place is fine. I just think interface or enum makes more sense for this instead of a class.